core/config/initializers/00_configuration.rb

```diff
@@ -156,14 +156,15 @@
   end
 
   fieldset 'Communication', namespaced: false do
+
     field 'Email From',
       type: :string,
-      default: 'noreply@example.com',
+      default: -> { "#{Workarea.config.site_name} <noreply@#{Workarea.config.host}>" },
```

Comment:
Is using a lambda actually supported? How did you test this?

Reply:
default is passed through directly to the mongoid field definition, so if its supported by mongoid its supported here.

Reply:
Sure is! I tested it locally on my dev machine, first by restarting my server and then destroying the existing admin config that I had. When I went to the admin configuration page, I saw "noreply@store.test" as my Email From.